packages/zoe/src/cleanProposal.js

```diff
@@ -88,6 +94,57 @@ export const cleanKeywords = keywordRecord => {
   return keywords;
 };
 
+const assertAfterDeadlineExitRule = exit => {
+  const expectedAfterDeadlineKeys = ['timer', 'deadline'];
+  assertKeysAllowed(expectedAfterDeadlineKeys, exit.afterDeadline);
+  assert(exit.afterDeadline.timer !== undefined, X`timer must be defined`);
+  assert(
+    typeof exit.afterDeadline.deadline === 'bigint' &&
+      isNat(exit.afterDeadline.deadline),
```

Comment:
It is funny that this stricter test corresponds to our likely resolution of the `NatValue` inconsistency. But even if so, we shouldn't test here that the deadline is a NatValue since that has nothing to do with the role of this number. So no change suggested here.